### PR TITLE
markdown: handle all cases of relative img path

### PR DESCRIPTION
**Pull request by getneil**

### CODE REVIEW

File: svelte/src/components/markdown/markdown.svelte [hunk 1]
```diff
@@ -28,8 +28,10 @@
   // TODO: support gitlab too
   const changeSrcIfNeeded = (element: HTMLElement | Element) => {
     const src = element.getAttribute("src");
-    if (src?.startsWith("/")) {
-      element.setAttribute("src", `https://raw.githubusercontent.com${src}`);
+    const isRelativePath = src?.match(/^[^/]+\/[^/].*$|^\/[^/].*$/);
```
Comment (getneil) [inline svelte/src/components/markdown/markdown.svelte:31]: just curious, it would be simpler to `!src.startsWith('http')` right?

Comment (ryanml) [reply svelte/src/components/markdown/markdown.svelte:31]: I think for our purposes that will be fine. Of course there are many other non http protocols/prefixes that can be used in absolute paths but for the sake of sniffing them out in README’s checking for http should do just fine :)

File: svelte/src/components/markdown/markdown.svelte [hunk 2]
```diff
@@ -28,8 +28,10 @@
...
+    if (isRelativePath) {
+      const url = new URL(src!, "https://github.com");
```
Comment (ryanml) [inline svelte/src/components/markdown/markdown.svelte:33]: should this be `const url = new URL(src, “https://github.com”);

Comment (getneil) [reply svelte/src/components/markdown/markdown.svelte]: i fixed that. it was possible for src to null

**Pull request merged**